src/components/main/Summary.tsx

```diff
@@ -0,0 +1,55 @@
+import Card from "../UI/Card"
+import axios from 'axios'
+import { useQuery } from '@tanstack/react-query'
+import { motion } from 'framer-motion'
+
+const Summary = () => {
+    const fetchDescription = async () => {
+        const res = await axios.get('/.netlify/functions/generate-description')
+        return res.data
+    }
+
+    const response = useQuery({
+        queryKey: ['description'],
+        queryFn: fetchDescription,
+        refetchOnMount: false,
+        refetchOnWindowFocus: false,
+    })
+
+    if (response.status === 'error') console.log(response)
+
+
+
+    return <Card className="grid grid-cols-2 items-center">
+
+        {/* sometimes the animation doesn't work, don't know why */}
+        <motion.div
```

Comment:
why not having 2-3 motion.divs depending of the state
{loading && <motion.div></motion.div>  } etc

Reply:
{!loading && <motion.div>I finished loading</motion.div>}

Reply:
and these 2 can be separate components

Reply:
I think I've figured out how to conditional rendering with framer motion - you need to use <AnimatePresence> as the parent component and wrap it around multiple <motion.div key=''> to have the animation effect on removing/adding components